TF_Application/Assets/TouchFree/Tooling/Scripts/Connection/MessageReceiver.cs

```diff
@@ -165,7 +165,8 @@ void CheckForAction()
                 if (actionQueue.TryPeek(out action))
                 {
                     // Stop shrinking the queue if we have a 'key' input event
-                    if (action.InputType != InputType.MOVE)
+                    if (action.InputType != InputType.MOVE ||
+                        action.InputType != InputType.NONE)
```

Comment:
For all of these new if statements, I believe we need &&, not || otherwise it won't skip _anything_ ?

Reply:
actions can only have one InputType, if we do && (AND) we'll never skip anything...?